diff --git a/node_modules/bookmarks-parser/lib/bookmarks-parser.js b/node_modules/bookmarks-parser/lib/bookmarks-parser.js
index c7be3c0..76f19c8 100644
--- a/node_modules/bookmarks-parser/lib/bookmarks-parser.js
+++ b/node_modules/bookmarks-parser/lib/bookmarks-parser.js
@@ -4,12 +4,12 @@ var
   async = require("async");
 
 // load parsers
-var files = fs.readdirSync(__dirname + "/parsers");
-files.forEach(function(f) {
-  if(!/^netscape\.js$/i.test(f)) {
-    parsers.push(require("./parsers/" + f));
-  }
-});
+//var files = fs.readdirSync(__dirname + "/parsers");
+//files.forEach(function(f) {
+//  if(!/^netscape\.js$/i.test(f)) {
+//    parsers.push(require("./parsers/" + f));
+//  }
+//});
 parsers.push(require("./parsers/netscape.js"));
 
 module.exports = exports = function(html, callback) {
