diff --git a/node_modules/bookmarks-parser/lib/bookmarks-parser.js b/node_modules/bookmarks-parser/lib/bookmarks-parser.js
index c7be3c0..64640bf 100644
--- a/node_modules/bookmarks-parser/lib/bookmarks-parser.js
+++ b/node_modules/bookmarks-parser/lib/bookmarks-parser.js
@@ -1,15 +1,15 @@
 var
   parsers = [],
-  fs = require("fs"),
+  //fs = require("fs"),
   async = require("async");
 
 // load parsers
-var files = fs.readdirSync(__dirname + "/parsers");
-files.forEach(function(f) {
-  if(!/^netscape\.js$/i.test(f)) {
-    parsers.push(require("./parsers/" + f));
-  }
-});
+//var files = fs.readdirSync(__dirname + "/parsers");
+//files.forEach(function(f) {
+//  if(!/^netscape\.js$/i.test(f)) {
+//    parsers.push(require("./parsers/" + f));
+//  }
+//});
 parsers.push(require("./parsers/netscape.js"));
 
 module.exports = exports = function(html, callback) {
diff --git a/node_modules/bookmarks-parser/lib/parsers/netscape.js b/node_modules/bookmarks-parser/lib/parsers/netscape.js
index 6e330bc..0daf106 100644
--- a/node_modules/bookmarks-parser/lib/parsers/netscape.js
+++ b/node_modules/bookmarks-parser/lib/parsers/netscape.js
@@ -1,7 +1,5 @@
-const { JSDOM } = require('jsdom')
-
+//const { JSDOM } = require('jsdom')
 exports.name = "netscape";
-
 exports.canParse = function(html, callback) {
   // first symbol most be <
   for(let i = 0; i < html.length; i++) {
@@ -25,9 +23,9 @@ exports.canParse = function(html, callback) {
 exports.parse = function(html, callback) {
   var rootFoldersRegEx = /^Menu|Unsorted|Toolbar$/i;
   try{
-    const { window } = new JSDOM(html)
+    let myparser = new DOMParser();
+    let bookmarksElement = myparser.parseFromString(html, "text/html");
     var result = [];
-
     // node is a DT element
     function _getNodeData( node ){
 
@@ -139,9 +137,7 @@ exports.parse = function(html, callback) {
       }
       return items;
     }
-
-    var dls = window.document.getElementsByTagName("DL");
-
+    var dls = bookmarksElement.getElementsByTagName("DL");
     if( dls.length > 0 ){
       callback(null, processDir( dls[0], 0 ));
     }
